static/docs/src.html

```diff
@@ -0,0 +1,193 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>
+      ReSpec Documentation
+    </title>
+    <script src=
+    "https://deploy-preview-3007--respec-pr.netlify.app/respec-w3c.js" class=
```

Comment:
half joking... but maybe we need a new documentation profile..

Reply:
w3c one should be fine I think.. though we don't use many of its features, and maintaining a new profile would be painful by itself.